flux/stdlib/influxdata/influxdb/from.go

```diff
@@ -6,97 +6,32 @@ import (
 	"github.com/influxdata/flux"
 	"github.com/influxdata/flux/codes"
 	"github.com/influxdata/flux/plan"
-	"github.com/influxdata/flux/runtime"
 	"github.com/influxdata/flux/stdlib/influxdata/influxdb"
 )
 
 const FromKind = "influxDBFrom"
 
-type FromOpSpec struct {
-	Bucket   string `json:"bucket,omitempty"`
-	BucketID string `json:"bucketID,omitempty"`
-}
-
-func init() {
```

Comment:
This logic was performing the same rewrite logic as the new `FromStorageRule` pushdown. I thought it'd be good to refactor into a 1:1 match with the 2.x code, so future back-/forward-ports have fewer differences to think through.